platform.md

```diff
@@ -146,94 +141,70 @@ The following is a non-exhaustive list of terms defined in the [OCI Distribution
 - The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`).
 
 The platform SHOULD ensure that:
 
 - The image config's `User` field is set to a user with a **DIFFERENT** user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID as the build image.
-- The image config's `Label` field has the label `io.buildpacks.stack.maintainer` set to the name of the stack maintainer.
-- The image config's `Label` field has the label `io.buildpacks.stack.homepage` set to the homepage of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.name` set to the name of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.version` set to the version of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.released` set to the release date of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.description` set to the description of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.metadata` set to additional metadata related to the stack.
+- The image config's `Label` field has the label `io.buildpacks.id` set to the target ID (e.g., "minimal") of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer.
+- The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image.
 
-### Mixins
+### Target Data
 
-A mixin name MUST only be defined by the author of its corresponding stack.
-A mixin name MUST always be used to specify the same set of changes.
-A mixin name MUST only contain a `:` character as part of an optional stage specifier.
+For both build images and run images, the platform MUST ensure that:
 
-A mixin prefixed with the `build:` stage specifier only affects the build image and does not need to be specified on the run image.
-A mixin prefixed with the `run:` stage specifier only affects the run image and does not need to be specified on the build image.
-A mixin WITHOUT a `build:` or `run:` prefix affects both the build and the run images.
+- The image config's `os` and `architecture` fields are set to valid identifiers as defined in the [OCI Image Specification](https://github.com/opencontainers/image-spec/blob/main/config.md).
```

Comment:
The OCI image spec uses "SHOULD" (as opposed to must) and defers to the go compiler's GOARCH list for valid architecture values.  However, we have a use-case for `*` architectures which doesn't exist in the go-compiler (example: a buildpack that is just shell scripts or batch scripts, which are inherently multi-arch).  Can we ammend this line to indicate that we're comfortable with a superset of the OCI spec's valid Architectures that also includes `*` arch?

Reply:
Does `*` arch mean missing arch, or literally `*`?

Reply:
I think it really means "any"

Reply:
Ah, now I understand (maybe). If we're talking about matching a buildpack that has stack `*` in buildpack.toml, then we might want to update the language around backwards-compat in the buildpack spec. I made a code suggestion here: https://github.com/buildpacks/spec/pull/336/files#r1142629644 - WDYT?